test/general/test_tag_dispatching.pass.cpp

```diff
@@ -0,0 +1,100 @@
+// -*- C++ -*-
```

Comment:
What the test does/checks?

Reply:
This text check that our `__is_backend_tag_v` and `__is_backend_tag_serial_v` works as designed.